alacritty_terminal/src/config/mod.rs

```diff
@@ -31,10 +31,35 @@ pub struct Config {
     /// Cursor configuration.
     pub cursor: Cursor,
 
+    /// Terminal specific settings.
+    pub terminal: Terminal,
+
     #[config(flatten)]
     pub pty_config: PtyConfig,
 }
 
+#[derive(ConfigDeserialize, Clone, Debug, PartialEq, Eq, Default)]
+pub struct Terminal {
+    // OSC 52 handling (clipboard handling).
+    pub osc52: Osc52,
+}
+
+#[derive(ConfigDeserialize, Clone, Debug, PartialEq, Eq, Default)]
+pub enum Osc52 {
+    /// The handling of the escape sequence is disabled.
+    Disabled,
+    /// Only copy sequence is accepted.
+    ///
+    /// This option is the default as a compromiss between entirely
+    /// disabling it (the most secure) and allowing `paste` (the less secure).
+    #[default]
+    OnlyCopy,
+    /// Only paste sequence is accepted.
+    OnlyPaste,
+    /// Both are accepted.
+    CopyPaste,
+}
```

Comment:
Arguably bitflags would be easier here inside of Alacritty, but I suppose the main reason for an enum was to allow easy (de)serialization?

Reply:
Yeah, and we don't have that many variants.